markdown/ch1.markdown

```diff
@@ -1,5 +1,12 @@
 # Elegant NumPy: The Foundation of Scientific Python
 
+> "[NumPy] is everywhere. It is all around us. Even now, in this very room.
+> You can see it when you look out your window or when you turn on your
+> television. You can feel it when you go to work... when you go to church...
+> when you pay your taxes."
```

Comment:
We don't use quotes when already in quote mode with `>`. See ch8 for a template!